### ISSUE TITLE
Please Help(cf Trying to access the properties #364)

### ISSUE THREAD

**Issue by contactzed8** [mapbox/mapbox-unity-sdk#612]
Hi.
I'm trying to access the property of my custom TileSet to enable it in textMeshPro.
I'm new on MapBox, could someone help me to do this. I tried to understand #364 but it's not clear enough for me.
Thanks a lot.

**Comment by jordy-isaac** [comment]
Hi @contactzed8 Can you let us know a bit more about your use case? What have you tried so far and where have you gotten stuck? Were you able to create your tileset in Mapbox studio and get it into Unity? Knowing more about where you are in the process and what exactly your end goal is will help us get you the right info.

**Comment by contactzed8** [comment]
Hi. Ok sorry I have to be clear :-)
To understand how I had to do, I created a simple tileset with one feature (point, layer "Takoma"). 
I attached a prefab in Unity to place a gameobject at this location. I would like to be able to access the data of the layer and make it visible when I want in a TextMeshPro object.
Screenshots attached :-)

<img width="810" alt="capture d ecran 2018-03-27 a 20 12 13" src="https://user-images.githubusercontent.com/35663042/37986693-6d4c867a-31fc-11e8-8d94-896ac4d92d10.png">
<img width="936" alt="capture d ecran 2018-03-27 a 20 22 23" src="https://user-images.githubusercontent.com/35663042/37986765-9d7efbac-31fc-11e8-8a11-68e40150f634.png">
